src/vs/workbench/api/node/extHostTypes.ts

```diff
@@ -1445,3 +1446,11 @@ export enum ConfigurationTarget {
 
 	WorkspaceFolder = 3
 }
+
+export class RelativePattern implements IRelativePattern {
+	base: string;
+
+	constructor(public pattern: string, base: vscode.WorkspaceFolder | string) {
```

Comment:
fugly, missing properties definitions like that

Reply:
Yeah, oversight, fixed via https://github.com/Microsoft/vscode/pull/34695/commits/328f82a9cef31b2c60523bf4d5060d3e039a316d